store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/AbstractAccumuloTwoSetSeededOperation.java

```diff
@@ -48,7 +47,6 @@ public AbstractAccumuloTwoSetSeededOperation(final Iterable<SEED_TYPE> seedsA, f
         this.setSeedsB(seedsB);
     }
 
-    @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.WRAPPER_OBJECT)
     @JsonGetter(value = "seedsB")
```

Comment:
The tests show it works without the annotation and the class property is already added in.